molecule/builder/tests/test_securedrop_deb_package.py

```diff
@@ -46,6 +46,21 @@ def get_deb_packages():
 
 deb_packages = get_deb_packages()
 
+def get_deb_tags():
+    """
+    Helper function to build array of package and tag tuples
+    for lintian.
+    """
+    deb_tags = []
+    
+    for deb in get_deb_packages():
+      for tag in securedrop_test_vars.lintian_tags:
+        deb_tags.append((deb, tag))
+
+    return deb_tags
+
+
+deb_tags = get_deb_tags()
```

Comment:
There are some flake8 violations in this file, which linting didn't catch. (I'll open a separate issue to fix the linting logic.) Could you run `flake8` against this file and ensure you see no violations?